skimage/filters/thresholding.py

```diff
@@ -133,9 +139,9 @@ def threshold_otsu(image, nbins=256):
 
     # Check if the image is multi-colored or not
     if image.min() == image.max():
-        raise TypeError("threshold_otsu is expected to work with images " \
-                        "having more than one color. The input image seems " \
-                        "to have just one color {0}.".format(image.min()))
+        raise ValueError("threshold_otsu is expected to work with images " \
```

Comment:
Change the expected exception in https://github.com/scikit-image/scikit-image/blob/master/skimage/filters/tests/test_thresholding.py#L168-L170 .